### PR TITLE
Populate from string

### PR DESCRIPTION
**Pull request by niels-qup**

### CODE REVIEW

File: reflection/populate_test.go [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+
+	if err != nil {
+		log.Print(err)
+		t.Fail()
+	}
+
+	if initializedStruct.Float64 != 9223372036854775800.12382136172 {
+		t.Fail()
+	}
+}
```
Comment (pcittadini) [inline reflection/populate_test.go:348]: I think test for bool type is missing. Or I don't see it anyhoo all the rest looks quite good to me.

**Review by daniel-qup — COMMENTED** [review]
It's also not immediately clear that the overflow tests for floats are missing.
But yes, it looks good. I'll take a closer look when I add tests for the csv reader.

**Comment by niels-qup** [comment]
> It's also not immediately clear that the overflow tests for floats are missing.
> But yes, it looks good. I'll take a closer look when I add tests for the csv reader.

Well that's correct, float overflow is a bit tricky for the Float64 format, i could possibly cook up something with string concat but i think we are strafing in the world of academic problems.

**Review by pcittadini — APPROVED** [review]
Approved on my end.

**Pull request merged**